It doesn't work with the unit tests, unfortunately:
```
        for sec in self.allseclist:
            for seg in sec:
                if i == idx:
                    command = cmd1 + pptype + cmd2
                    stim = eval(command, locals(), globals())
                    for key, value in kwargs.items():
                        try:
                          itr = iter(value)
                        except TypeError:
                            setattr(stim, key, value)
                        else:
>                           for i, v in itr:
E                           TypeError: cannot unpack non-iterable int object
```

--- a/LFPy/cell.py
+++ b/LFPy/cell.py
@@ -655,7 +655,7 @@ class Cell(object):
                     stim = eval(command, locals(), globals())
                     for key, value in kwargs.items():
                         try:
-                          itr = iter(value)
+                          itr = enumerate(iter(value))
                         except TypeError:
                             setattr(stim, key, value)
                         else:

